core/src/main/scala/kafka/network/RequestChannel.scala

```diff
@@ -365,7 +369,7 @@ class RequestMetrics(name: String) extends KafkaMetricsGroup {
     else
       None
 
-  private val errorMeters = mutable.Map[Errors, ErrorMeter]()
+  val errorMeters = mutable.Map[Errors, ErrorMeter]()
```

Comment:
I think it's good to avoid exposing mutable collections publicly. Typically, one can expose the operations required instead.